src/locales/default.json

```diff
@@ -235,7 +236,8 @@
     "uploading": "Uploading image",
     "markdown": "Styling with Markdown is supported",
     "errorGettingSnapshot": "We encountered an error while fetching the snapshot block number which is needed to calculate your voting power. Please try again later.",
-    "errorTimeInPast": "Entered time is in the past. Please select a future time."
+    "errorTimeInPast": "Entered time is in the past. Please select a future time.",
+    "errorSpaceHibernated": "This space has been hibernated, and proposals creation has been disabled until the space is reactivated by an admin."
```

Comment:
Need separate warning for admins, maybe a space wide banner with something like:

"This space has been disabled because there hasn't been a proposal in the past 6 months. Once you create a proposal, it will be reactivated."